charts/zeebe-benchmark/Chart.yaml-e

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: zeebe-benchmark
+description: A Helm chart for Zeebe benchmarks
+type: application
+version: 0.2.3
+appVersion: "8.2.5"
```

Comment:
❓ Was this file expected to be added?

Reply:
It was an error that I did in changing the scrip from linux to run MacOS! I will create an issue to document the work-around to use this in Mac.